samples/Exporters/Web/Startup.cs

```diff
@@ -36,6 +37,11 @@ public void ConfigureServices(IServiceCollection services)
                 }
             });
 
+
+            OpenTelemetrySdk.EnableOpenTelemetry(
+                (builder) => builder.AddActivitySource("Microsoft.AspNetCore")
+                .UseConsoleActivityExporter(opt => opt.DisplayAsJson = opt.DisplayAsJson));

```

Comment:
for easily trying this out locally, i changed to use consoleexporter.
The Instrumention(Adapter) hook to OpenTelemetryBuilder is not implemented in this draft PR.